DemoServer/client/src/utils/cookies/CookieJar.ts

```diff
@@ -35,6 +35,11 @@ export class CookieJar {
                 return null;
         }
     }
+    
+    public static isTrackingDisabledCookieSet(): boolean {
```

Comment:
The function name is a bit misleading. It will return false even if the cookie **is set** to `false`. I'd rename it to `isTrackingDisabled()`.

Reply:
Done.